src/clock-weather-card.ts

```diff
@@ -183,35 +183,38 @@ export class ClockWeatherCard extends LitElement {
   private renderForecast(): TemplateResult[] {
     const weather = this.getWeather();
     const currentTemp = roundIfNotNull(this.getCurrentTemperature());
-    const days = this.config.forecast_days;
-    const temperatueUnit = weather.attributes.temperature_unit;
+    const items = this.config.forecast_days;
+    const hourly = this.config.forecast_hourly;
+    const temperatureUnit = weather.attributes.temperature_unit;
 
-    const dailyForecasts = this.extractDailyForecasts(weather.attributes.forecast, days);
+    const forecasts = this.extractForecasts(weather.attributes.forecast, items, hourly);
 
-    const minTemps = dailyForecasts.map((f) => f.templow);
-    const maxTemps = dailyForecasts.map((f) => f.temperature);
+    const minTemps = forecasts.map((f) => f.templow);
+    const maxTemps = forecasts.map((f) => f.temperature);
     if (currentTemp !== null) {
       minTemps.push(currentTemp);
       maxTemps.push(currentTemp);
     }
     const minTemp = Math.round(min(minTemps));
     const maxTemp = Math.round(max(maxTemps));
     
-    const gradientRange = this.gradientRange(minTemp, maxTemp, temperatueUnit);
-    return dailyForecasts.map((forecast) => safeRender(() => this.renderForecastDay(forecast, gradientRange, minTemp, maxTemp, currentTemp)));
+    const gradientRange = this.gradientRange(minTemp, maxTemp, temperatureUnit);
+    return forecasts.map((forecast) => safeRender(() => this.renderForecastItem(forecast, gradientRange, minTemp, maxTemp, currentTemp, hourly)));
   }
 
-  private renderForecastDay(forecast: MergedWeatherForecast, gradientRange: Rgb[], minTemp: number, maxTemp: number, currentTemp: number | null): TemplateResult {
-    const dayText = this.localize(`day.${new Date(forecast.datetime).getDay()}`);
+  private renderForecastItem(forecast: MergedWeatherForecast, gradientRange: Rgb[], minTemp: number, maxTemp: number, currentTemp: number | null, hourly: boolean): TemplateResult {
+    const twelveHour = this.getTimeFormat() === '12';
+    const displayText = !hourly ? this.localize(`day.${new Date(forecast.datetime).getDay()}`) : format(new Date(forecast.datetime), !twelveHour ? 'HH:mm' : 'h:mm aa');
```

Comment:
boxing via `new Date(...)` not required, as `forecast.datetime` is already of type `Date`

Reply:
additionally, to not have the formatting logic (`HH:mm` vs `h:mm aa`) in two different locations redundantly, you could extend the existing method `this.time()` for formatting:
```
  private time(date: Date = this.currentDate): string {
    return format(date, this.getTimeFormat() === '24' ? 'HH:mm' : 'h:mm aa');
  }
```

then you would simplify the whole line to: 
`const displayText = !hourly ? this.localize(`day.${forecast.datetime.getDay()}`) : this.time(forecast.datetime);`

Reply:
Addded it, thanks for the heads up